exercises/J-array-get-set/exercises2.js

```diff
@@ -7,7 +7,7 @@
 */
 
 var numbers = [1, 2, 3]; // Don't change this array literal declaration
-
+numbers [3] = 4; 
```

Comment:
There is a better way of doing this. What if the length of the array keeps on changing dynamically? How would you assign number 4 to the end of the array?